lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -234,28 +234,33 @@ defmodule ExUnit.Callbacks do
   end
 
   @doc false
-  def __merge__(_mod, context, :ok) do
+  def __merge__(mod, context, value) do
+    __merge__(mod, context, value, value)
+  end
+
+  @doc false
+  def __merge__(_mod, context, :ok, _original_value) do
```

Comment:
That's `__merge__/3` though, I was referring to `__merge__/4` :) It looks like we can turn `__merge__/4` into `defp merge/4`.